impl/authentication/context.go

```diff
@@ -14,24 +14,25 @@
 
 // Package authentication implements authentication mechanisms.
 //
-// The Transparent Key Server is designed to be used by identity providers -
+// Key Transparency is designed to be used by identity providers -
 // IdP in OAuth parlance.  OAuth2 Access Tokens may be provided as
 // authentication information, which can be resolved to user information and
 // associated scopes on the backend.
 package authentication
 
-import (
-	"context"
-	"errors"
-)
+import "context"
 
-var (
-	// ErrMissingAuth occurs when authentication information is missing.
-	ErrMissingAuth = errors.New("auth: missing authentication header")
-)
+// SecurityContext is the auth value stored in the Contexts.
+type SecurityContext struct {
+	Email string
+}
+
+// securityContextKey identifies ValidatedSecurity within context.Context.
+var securityContextKey struct{}
 
-// Authenticator provides services to authenticate users.
-type Authenticator interface {
-	// ValidateCreds authenticate the information present in ctx.
-	ValidateCreds(ctx context.Context) (*SecurityContext, error)
+// FromContext returns a ValidatedSecurity from the current context.
```

Comment:
This doc wasn't updated for the rename (x2 mentions)

Reply:
Thanks.